crossbar/adapter/rest/common.py

```diff
@@ -96,8 +96,12 @@ def _deny_request(self, request, code, reason, **kwargs):
         """
         Called when client request is denied.
         """
+        if "cb_log_id" not in kwargs.keys():
```

Comment:
from slack, my reason for picking this was that the `cb_` namespacing means we might not conflict with anything, but that is not a big deal, and `log_category` is easier to type/remember.